modules/op-differences-between-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="differences-between-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Differences between non-versioned and versioned cluster tasks
+
+Non-versioned and versioned cluster tasks have different naming conventions. And, the {pipelines-title} Operator upgrades them differently.
+
+.Differences between non-versioned and versioned cluster tasks
+[options="header"]
+|===
+
+| | Non-versioned cluster task | Versioned cluster task
+
+| Nomenclature  
+| The non-versioned cluster task variant only contains the name of the cluster task. For example, the name of the non-versioned variant of the Buildah cluster task installed with Operator v1.7 is `buildah`.  
```

Comment:
I have removed all instances of variants except one in the `managing-nonversioned-and-versioned-cluster-tasks.adoc` assembly. I left it deliberately because I couldn't think of an alternate framing of the sentence with the same meaning.